qgreenland/util/luigi/tasks/pipeline.py

```diff
@@ -158,14 +161,23 @@ def output(self):
 
     def run(self):
         logger.info(f'Creating {PROJECT} package: {self.output().path} ...')
+        input_path = Path(self.input().path)
+        output_path = Path(self.output().path)
 
         # Create the archive from the symlinked dir.
-        tmp_name = f'{TMP_DIR}/final_archive'
-        shutil.make_archive(tmp_name, 'zip', WIP_DIR, self.input().path)
-        Path(f'{tmp_name}.zip').rename(self.output().path)
+        tmp_fp = TMP_DIR / 'final_archive.zip'
+        tmp_name = f'{tmp_fp.parent}/{tmp_fp.stem}'
+
+        shutil.make_archive(
+            tmp_name,
+            'zip',
+            WIP_DIR,
+            input_path.relative_to(WIP_DIR),
+        )
+        tmp_fp.rename(output_path)
```

Comment:
TODO: Why is the rename step still here? Didn't this change eliminate the need to rename?